doc/conf_ref.rst

```diff
@@ -222,6 +222,19 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     RPM debug scriptlet output level. One of: ``critical``, ``emergency``,
     ``error``, ``warn``, ``info`` or ``debug``. Default is ``info``.
 
+.. _search_enabled_metadata-label:
+
+``search_enabled_metadata``
```

Comment:
I think this should be default (i.e. without conf option). Probably there should be option `allow_use_enabled_metadata` or such which would allow doing depsolving using those repos or smth like that.